test/test_pipeline_baseline.py

```diff
@@ -97,17 +109,114 @@ def _test_baseline(task_name, test_name, language, output_dir, extra_kwargs,
     check_calls_match(expected_subprocess_call, mock_call.mock_calls)
 
 
+@mock.patch('pipeline.utils.task_utils.runGradleTask')
+@mock.patch('subprocess.call')
+@mock.patch('subprocess.check_call')
+@mock.patch('os.chdir')
+def _test_cli_baseline(pipeline_name, config, pipeline_kwargs, baseline,
```

Comment:
What exactly is this testing? It looks a lot like `_test_baseline` -- is there some way we can factor out the common code?